src/app/pages/ArticlePage/PagePromoSections/RelatedContentSection/index.test.jsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { ServiceContextProvider } from '#app/contexts/ServiceContext';
+import { ToggleContextProvider } from '#contexts/ToggleContext';
+import * as clickTracking from '#hooks/useClickTrackerHandler';
+import * as viewTracking from '#hooks/useViewTracker';
+import RelatedContent from '.';
+import {
+  RelatedContentList,
+  RelatedContentSingleItem,
+  RelatedContentCustomLabel,
+} from './fixture';
+
+// eslint-disable-next-line react/prop-types
+const RelatedContentComponent = ({ fixtureData, service = 'mundo' }) => (
+  <ServiceContextProvider service={service}>
+    <ToggleContextProvider>
+      <RelatedContent content={fixtureData} />
+    </ToggleContextProvider>
+  </ServiceContextProvider>
+);
+
+describe('Optimo Related Content Promo', () => {
+  it('should return null if no data is passed', () => {
+    const { container } = render(<RelatedContentComponent fixtureData={{}} />);
+    expect(container).toBeEmptyDOMElement();
+  });
+
+  it('should render Related Content Ul when given More than one Related Content', () => {
+    const { getAllByRole, container } = render(
+      <RelatedContentComponent fixtureData={RelatedContentList} />,
+    );
+    const listItems = getAllByRole('listitem');
+    const list = container.querySelector('ul');
+    expect(listItems.length).toBe(3);
+    expect(list).toBeInTheDocument();
+  });
+
+  it('should render custom label text if provided ', () => {
+    const { getByText } = render(
+      <RelatedContentComponent fixtureData={RelatedContentCustomLabel} />,
+    );
+    const customLabel = getByText('Related content block');
+
+    expect(customLabel).toBeInTheDocument();
+  });
+
+  it('should render a default title if translations are not available', () => {
+    const { getByText } = render(
+      <RelatedContentComponent
+        fixtureData={RelatedContentList}
+        service="news"
+      />,
+    );
+
+    const label = getByText(`Related content`);
+    expect(label).toBeInTheDocument();
+  });
+
+  it('should have a "region" role (a11y)', () => {
+    const { getByRole } = render(
+      <RelatedContentComponent fixtureData={RelatedContentList} />,
+    );
+    const region = getByRole('region');
+    expect(region).toBeInTheDocument();
+  });
+
+  it("should have a section labelled-by the section label's id (a11y)", () => {
```

Comment:
I think `(a11y)` is not necessary as it's obvious this is testing accessibility. This will become inconsistent when more devs write components and don't use `(a11y)`

Reply:
Removed